### DISCUSSION TITLE
Big Sur Adobe Animate file access errors in 33.1.1.575

### DISCUSSION THREAD

**Discussion by elemdeso** [airsdk/Adobe-Runtime-Support#1071] [Q&A]
33.1.1.444 was the last SDK I was able to successfully publish from. Since then, on 544 and 575, I've had errors accessing bin/adl and files in lib. I'm on the latest Big Sur, XCode and Animate 2021 on an Intel MacMini. Thing is when I replaced lib/adt.jar that comes with v575 with the old version from 444 I can now test and publish again. What's changed in adt.jar since 33.1.444?

**Comment by elemdeso** [comment]
Sorted. The SDK folder inside Adobe Animate used to be "AIR 33.1". Now it has to be "AIR33.1" - wrong name, bad path.

**Reply by ajwfrost** [reply]
Thanks @elemdeso  - I can see how some tools may go missing if there's a space in the file path, if they're not properly escaped .. but I would wonder how that affected the "adl" binary, and in particular how it ended up working if you used the 33.1.1.444 version of ADT. But there are a number of changes in ADT so it's possible something there is not coping with spaces in paths: can you please clarify what it was that didn't work (a particular type of publish command? was this for a desktop app or for a mobile one? etc) - and ideally, if you can find the SDK folder and edit the `lib/adt.cfg` file to uncomment `DebugOut=true`, then run it (with the `AIR 33.1` folder name so that it fails) and send us the file that it should generate at `~/adt.log` .. we may then be able to see what was being run that doesn't cope with the space in the path...

thanks

**Reply by elemdeso** [reply]
HI @ajwfrost Thanks for picking up on this. it's an iPad only app. At first I thought maybe I just got mixed up but I've been back through the steps and it's true. If I install 575 in a top level folder named (as before) 'AIR 33.1' inside 'Adobe Animate 2021', it fails. I get the two warnings attached:
desktop testing (dev cert) - 
![failed-test](https://user-images.githubusercontent.com/44811347/129593956-549ad742-730e-4b3f-bcce-ebdf8819cb49.png)
App Store publishing (distrib cert) - 
![failed-publish](https://user-images.githubusercontent.com/44811347/129593999-0e5f0d38-5c17-4b26-9dcd-56d9ab5d6afb.png)

If I then replace lib/adt.jar with the one from 444 it tests and publishes fine. Or if I change the top level folder to 'AIR33.1' it tests and publishes fine with the 575 lib/adt.jar.

Here's the log. The first lines are for the fail (nothing there) and the rest for the successful publish -
[adt.log](https://github.com/airsdk/Adobe-Runtime-Support/files/6993781/adt.log)

**Reply by itlancer** [reply]
@elemdeso I saw the same error earlier. Seems that AIR SDK for macOS contains AIR SDK for Windows. I downloaded AIR SDK for macOS again from https://airsdk.harman.com/download and now it works fine.
